tests/containers/podman_quadlet.pm

```diff
@@ -12,34 +12,48 @@ use testapi;
 use serial_terminal qw(select_serial_terminal);
 use version_utils qw(package_version_cmp);
 use Utils::Systemd qw(systemctl);
+use JSON;
 
-sub check_unit_states {
-    my $expected = shift // 'generated';
-    validate_script_output('systemctl --no-pager is-enabled sleeper.service', qr/$expected/, proceed_on_failure => 1);
-    validate_script_output('systemctl --no-pager is-enabled sleeper-volume.service', qr/$expected/, proceed_on_failure => 1);
-}
+my $unit_prefix = '/etc/containers/systemd';
```

Comment:
```suggestion
my $quadlet_dir = '/etc/containers/systemd';
```
I think the variable name `quadlet_dir` would be more descriptive than `unit_prefix`. What do you think? :thinking: